### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Add shortcut name to my shortcuts

### DISCUSSION THREAD

**Discussion by Khesir** [dailydotdev/daily#1276] [Feature Requests]
### Description

Hi! I really like using daily.dev and I considering this as my new tab page for now on.

Can you add a name option just like google shortcuts to my shortcuts?
this is kinda annoying to see links e.g website..c..

![image](https://github.com/dailydotdev/daily/assets/87099653/cf067744-9826-4180-ada4-f699ef2b9cea)

### Solution Proposed

I like the idea of adding it all at once, can you maybe add another field for name?

![image](https://github.com/dailydotdev/daily/assets/87099653/cbd5559b-6c1e-4b29-8407-1ca0281ab2cf)

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @khesir,

Thanks for the suggestion. We will consider it 🙏
